engine/installation/linux/centos.md

```diff
@@ -1,232 +1,234 @@
+    > or `yum update` command will always install the highest possible version,
+    > which will almost certainly be an unstable one.
 
     ```bash
-    $ sudo systemctl enable docker.service
+    $ sudo yum-config-manager --set-enabled docker-testing
     ```
 
-6. Start the Docker daemon.
+    You can disable the `testing` repository by running the `yum-config-manager`
+    command with the `--set-disabled` flag. To re-enable it, use the
+    `--set-enabled` flag. The following command disables the `testing`
+    repository.
 
     ```bash
-    $ sudo systemctl start docker
+    $ sudo yum-config-manager --set-disabled docker-testing
     ```
 
-7. Verify `docker` is installed correctly by running a test image in a container.
-
-        $ sudo docker run --rm hello-world
-
-        Unable to find image 'hello-world:latest' locally
-        latest: Pulling from library/hello-world
-        c04b14da8d14: Pull complete
-        Digest: sha256:0256e8a36e2070f7bf2d0b0763dbabdd67798512411de4cdcf9431a1feb60fd9
-        Status: Downloaded newer image for hello-world:latest
-
-        Hello from Docker!
-        This message shows that your installation appears to be working correctly.
-
-        To generate this message, Docker took the following steps:
-         1. The Docker client contacted the Docker daemon.
-         2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
-         3. The Docker daemon created a new container from that image which runs the
-            executable that produces the output you are currently reading.
-         4. The Docker daemon streamed that output to the Docker client, which sent it
-            to your terminal.
-
-        To try something more ambitious, you can run an Ubuntu container with:
-         $ docker run -it ubuntu bash
-
-        Share images, automate workflows, and more with a free Docker Hub account:
-         https://hub.docker.com
-
-        For more examples and ideas, visit:
-         https://docs.docker.com/engine/userguide/
-
-If you need to add an HTTP Proxy, set a different directory or partition for the
-Docker runtime files, or make other customizations, read our Systemd article to
-learn how to [customize your Systemd Docker daemon options](../../admin/systemd.md).
+#### Install Docker
 
-### Install with the script
-
-1. Log into your machine as a user with `sudo` or `root` privileges.
-
-2. Make sure your existing packages are up-to-date.
+1.  Update the `yum` package index.
 
     ```bash
-    $ sudo yum update
+    $ sudo yum -y check-update
     ```
 
-3. Run the Docker installation script.
+2.  Install the latest version of Docker, or go to the next step to install a
+    specific version.
 
     ```bash
-    $ curl -fsSL https://get.docker.com/ | sh
+    $ sudo yum -y install docker-engine
     ```
 
-    This script adds the `docker.repo` repository and installs Docker.
+    > **Warning**: If you have both stable and unstable repositories enabled,
```

Comment:
Once during enabling the repos, once when actually making the decision about what version to install.